packages/lib/src/toolbar/controls/Selector/Selector.module.css

```diff
@@ -108,10 +111,12 @@
 }
 
 .option:hover,
+.option[data-active-item],
```

Comment:
https://ariakit.org/guide/styling#css-selectors

Reply:
`.option:hover` and `.option:focus` will probably be removed once I finish refactoring the other menus.